requirements_test_all.txt

```diff
@@ -81,7 +81,7 @@ hbmqtt==0.9.2
 holidays==0.9.5
 
 # homeassistant.components.frontend
-home-assistant-frontend==20180803.0
+home-assistant-frontend==20180804.0
```

Comment:
This shouldn't change.

Reply:
The build failed and said that I need to re-run requirements script. This was result. Otherwise build did not work for some reason ...

Reply:
I tried re-running gen_requirements_all.py. If I back out those changes, the build will fail again. What do you suggest I do?